### DISCUSSION TITLE
task_group in templates

### DISCUSSION THREAD

**Discussion by Pocoder** [apache/airflow#25087] [Q&A]
How to get params (or default_args['params']) of current task group context in template? `{{ task.task_group.params }}` falls with: `jinja2.exceptions.UndefinedError: 'weakproxy object' has no attribute 'params'`

**Comment by potiuk** [comment]
What makes you think task_group has params?

**Reply by Pocoder** [reply]
It would be logically if dag, task group and task have params field. If task group has no params and only add default_args into its tasks in time of "parsing" dag file It isn't possible to do what I want now, right?

**Reply by potiuk** [reply]
Yeaah maybe it's a good idea to add something like that. But I was rather asking what made you think it is there (none of the docs suggest it's there and I could not find it). What made you think it is there other than your impression that it should be there? Did yo actually look at the docs and had wrong impression? Is our documents not clear about it? 

If you think it's a worthwile addition - please by all means propese a featuer request (it's small enough to not require bigger discussion) or - much better - propose a pull request implementing it. In order for something like that to be implemented, someone would have to pick an interest in such feature and implement it - Airlfow is created by almost 2200 contributors (most of them users like you) who added features  when they saw them missing - you can follow their footsteps and do the same. 

In any case if you add such a feature idea we can mark it as good first issue and hope someone will implement it actually.

**Reply by Pocoder** [reply]
This assumption appeared more because of what I wanted to do and not what I know is possible from docs. 

I will make feature request in the near future or will implement it myself.

**Reply by potiuk** [reply]
> I will make feature request in the near future or will implement it myself.

Cool. Looking forward to it.
